src/popover/__tests__/popover-autofocus.e2e.js

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright (c) Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+/* eslint-env node */
+/* eslint-disable flowtype/require-valid-file-annotation */
+
+const {mount} = require('../../../e2e/helpers');
+
+const isActiveEl = async (page, selector) => {
+  // eslint-disable-next-line cup/no-undef
+  const activeEl = await page.evaluateHandle(() => document.activeElement);
+  const selectedEl = await page.$(selector);
+  const equal = await page.evaluate(
+    (e1, e2) => e1 === e2,
+    activeEl,
+    selectedEl,
+  );
+  activeEl.dispose();
+  return equal;
+};
+
+const inputSelector = 'input';
+
+describe('popover', () => {
+  it('contents are auto-focused but not focus-locked', async () => {
```

Comment:
Do you mind adding a test for the new scenario file asserting that when the popover opens, the input selector is focused and after tab key press, the input selector is still focused?